### DISCUSSION TITLE
Horizontal button stack: buttons mounted on mobile

### DISCUSSION THREAD

**Discussion by agak79** [Clooos/Bubble-Card#403] [Questions about config, custom styles and templates]
Hi

I have configured my pop up cards an horizontal button stack with 3 buttons to complete my mobile dashboard, but the buttons are mounted on top of each other
![image](https://github.com/Clooos/Bubble-Card/assets/157421914/121d85e5-ba76-41c2-9e03-6bcbe3c0ca1b)

Is there any way to separate them?

**Comment by MrBearPresident** [comment]
Did you solve this?
Can this be closed?

**Reply by agak79** [reply]
No, i didnt

**Reply by Menelao147** [reply]
Could you share your code?
